src/OpenLoco/src/Drawing/SoftwareDrawingContext.cpp

```diff
@@ -1280,13 +1256,15 @@ namespace OpenLoco::Drawing
 
             _currentFontSpriteBase = Font::medium_bold;
             uint16_t width = getStringWidth(buffer);
+            auto point = origin;
+            point.x -= width;
 
-            drawString(rt, x - width, y, colour, buffer);
+            drawString(rt, point, colour, buffer);
 
             // Draw underline
-            drawRect(rt, x - width, y + 11, width, 1, _textColours[1], RectFlags::none);
+            drawRect(rt, origin.x - width, origin.y + 11, width, 1, _textColours[1], RectFlags::none);
             if (_textColours[2] != 0)
-                drawRect(rt, x - width, y + 12, width, 1, _textColours[2], RectFlags::none);
+                drawRect(rt, origin.x - width, origin.y + 12, width, 1, _textColours[2], RectFlags::none);
```

Comment:
same here but `point.y + 12`

Reply:
Then all of this is identical to drawStringLeftUnderline. So you could even just extract out the common function

Reply:
Yeah, I suppose. Eventually we should probably refactor things to just have the single `drawString` method with alignment and font as parameters. OpenRCT2 does something similar, correct?

Reply:
> single drawString method with alignment and font as parameters

this sounds like a good idea for me